Change MyBinder links to VEDA Binder

j08lue opened the pull request:
Closes

- https://github.com/NASA-IMPACT/veda-docs/issues/187

By

- [x] Change links to MyBinder to point at VEDA Binder
- [x] Add section explaining what Binder and VEDA Binder is

wildintellect commented on nasa-veda-platform/adding-content/docs-and-notebooks.qmd:
```diff
@@ -67,7 +67,7 @@ Something like: public.ecr.aws/nasa-veda/pangeo-notebook:60b023fba2ca5f9e19d285c
 To give the notebooks a standard look and feel we typically include the following sections:
 
-- **Run this Notebook**: The section explains how to run the notebook locally, on VEDA JupyterHub or on [mybinder](https://mybinder.org/). There are several examples of what this section can look like in the template notebooks.
+- **Run this Notebook**: The section explains how to run the notebook locally, on [VEDA JupyterHub](https://hub.openveda.cloud/), on [VEDA Binder](https://binder.openveda.cloud/), or on the public [MyBinder](https://mybinder.org/). There are several examples of what this section can look like in the template notebooks.
```
should the jupyterhub link be an internal reference to https://docs.openveda.cloud/nasa-veda-platform/scientific-computing/ ?
Or something that explains the Hub? Most people with hub access won't need the binder links.

wildintellect replied:
I'm also concerned about link to https://binder.openveda.cloud if we don't have guardrails, since it might be abused by non-veda related work. @yuvipanda

j08lue replied:
We started discussing the implications of public access on Slack. Let's continue the discussion there but merge this docs change for now.

wildintellect commented on nasa-veda-platform/scientific-computing/index.qmd:
```diff
@@ -15,12 +15,21 @@ VEDA scientific computing services provide scientists the resources they need to
 
 VEDA promotes the use of JupyterHub environments for interactive data science. JupyterHub enables you to analyze massive archives of Earth science data in the cloud in an interactive environment that alleviates the complexities of managing compute resources (virtual machines, roles and permissions, etc).
 
+### Limited access JupyterHub service
```
swap Limited for a more positive term, like Sponsored or ?

j08lue replied:
```suggestion
### VEDA Sponsored JupyterHub service
```

wildintellect commented on notebooks/index.qmd:
```diff
@@ -31,7 +31,7 @@ The **Quickstarts** examples are further divided into two sections, which you ca
 ## How to run
 
-Every notebook contains information about how to run it. Some can run on [mybinder](https://mybinder.org/) and all can run on the VEDA JupyterHub. See [VEDA Analytics JupyterHub Access](https://nasa-impact.github.io/veda-docs/veda-jh-access.html) for information about how to gain access.
+Every notebook contains information about how to run it. Some can run on [Binder](https://binder.openveda.cloud/) and all can run on the [VEDA JupyterHub](https://hub.openveda.cloud/). See [VEDA Analytics JupyterHub Access](https://nasa-impact.github.io/veda-docs/veda-jh-access.html) for information about how to gain access.
```
should we pick a different term from Binder, that doesn't really explain what it is - but rather the technical tool that allows it. Maybe something like Trial or Test

j08lue replied:
Called it VEDA Trial Hub for now

wildintellect reviewed (commented):
All the links seem to be good (I tested some), made some comments around the descriptions, nothing is a blocker.

j08lue commented on nasa-veda-platform/adding-content/docs-and-notebooks.qmd:
```diff
@@ -67,7 +67,7 @@ Something like: public.ecr.aws/nasa-veda/pangeo-notebook:60b023fba2ca5f9e19d285c
 To give the notebooks a standard look and feel we typically include the following sections:
 
-- **Run this Notebook**: The section explains how to run the notebook locally, on VEDA JupyterHub or on [mybinder](https://mybinder.org/). There are several examples of what this section can look like in the template notebooks.
+- **Run this Notebook**: The section explains how to run the notebook locally, on [VEDA JupyterHub](https://hub.openveda.cloud/), on [VEDA Binder](https://binder.openveda.cloud/), or on the public [MyBinder](https://mybinder.org/). There are several examples of what this section can look like in the template notebooks.
```
```suggestion
- **Run this Notebook**: The section explains how to run the notebook locally, on [VEDA JupyterHub](https://docs.openveda.cloud/nasa-veda-platform/scientific-computing/), on [VEDA Ephemeral Hub](https://binder.openveda.cloud/), or on the public [MyBinder](https://mybinder.org/). There are several examples of what this section can look like in the template notebooks.
```

j08lue replied:
```suggestion
- **Run this Notebook**: The section explains how to run the notebook locally, on [VEDA JupyterHub](https://hub.openveda.cloud/), on the [VEDA Trial Hub](https://binder.openveda.cloud/), or on the public [MyBinder](https://mybinder.org/). There are several examples of what this section can look like in the template notebooks.
```

j08lue commented on notebooks/index.qmd:
```diff
@@ -31,7 +31,7 @@ The **Quickstarts** examples are further divided into two sections, which you ca
 ## How to run
 
-Every notebook contains information about how to run it. Some can run on [mybinder](https://mybinder.org/) and all can run on the VEDA JupyterHub. See [VEDA Analytics JupyterHub Access](https://nasa-impact.github.io/veda-docs/veda-jh-access.html) for information about how to gain access.
+Every notebook contains information about how to run it. Some can run on [Binder](https://binder.openveda.cloud/) and all can run on the [VEDA JupyterHub](https://hub.openveda.cloud/). See [VEDA Analytics JupyterHub Access](https://nasa-impact.github.io/veda-docs/veda-jh-access.html) for information about how to gain access.
```
```suggestion
Every notebook contains information about how to run it. Some can run on [VEDA Trial Hub](https://binder.openveda.cloud/) and all can run on the [VEDA JupyterHub](https://hub.openveda.cloud/). See [VEDA Analytics JupyterHub Access](https://nasa-impact.github.io/veda-docs/veda-jh-access.html) for information about how to gain access.
```

j08lue commented on nasa-veda-platform/scientific-computing/index.qmd:
```diff
@@ -15,12 +15,21 @@ VEDA scientific computing services provide scientists the resources they need to
+### Public VEDA Binder service
+
+Some of our tutorial notebooks do not require direct data access or substantial CPU or memory resources. 
+These can also be executed interactively in the open access [VEDA Binder](https://binder.openveda.cloud/) service, a copy of [MyBinder](https://mybinder.org/). 
```
```suggestion
These can also be executed interactively in the open access [VEDA Trial Hub (Binder)](https://binder.openveda.cloud/) service, a copy of [MyBinder](https://mybinder.org/). 
```

j08lue commented:
@wildintellect I addressed your comments and recommend that we merge this for now and follow up on the discussion re Binder (now "VEDA Trial Hub") elsewhere.
